src/Microsoft.DotNet.Wpf/src/PresentationFramework/System/Windows/Controls/DataGridLengthConverter.cs

```diff
@@ -267,7 +260,7 @@ private static DataGridLength ConvertFromString(string s, CultureInfo cultureInf
                     (unit == DataGridLengthUnitType.Pixel) || DoubleUtil.AreClose(unitFactor, 1.0),
                     "unitFactor should not be other than 1.0 unless the unit type is Pixel.");
 
-                ReadOnlySpan<char> valueString = goodString.AsSpan(0, strLen - strLenUnit);
+                ReadOnlySpan<char> valueString = valueSpan.Slice(0, valueSpan.Length - strLenUnit);
```

Comment:
Yes we would now call double.Parse with the non-lowercased value. I chose to unify all converters to do the same because of this comment here which explains why we should double.Parse with the non-lowercased value (Though the comment seems obsolete because I made some tests and I checked the code of double.Parse and the case of the value in double.Parse shouldn't matter): https://github.com/dotnet/wpf/pull/8622/files/8ea913b2f0e29f8d82d2bcd23dcc555996abc7d7#diff-c16c7b22fe582e51d67bbddff73c7ed62887fde21bb575147ee6c62c09c9b6bcL207-L210

Reply:
Yeah, testing English based alphabets, figured out that the case should not have any affect on double.Parse, but wanted to point it out as I wasn't sure due to the involvement of different types of culture here.